src/components/Book.js

```diff
@@ -29,7 +29,7 @@ const Book = (props) => {
 
 Book.propTypes = {
   bookItem: PropTypes.exact({
-    id: PropTypes.number,
+    item_id: PropTypes.number,
```

Comment:
- [ ] as the previous CR already mentioned please change line 32 to:
```
item_id: PropTypes.string,
```
because it is throwing an error at the moment.

Reply:
Fixed🚀